supabase/migrations/20240505115913_releases.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE public.releases (
+     id UUID NOT NULL PRIMARY KEY,
+     project_id UUID NOT NULL REFERENCES public.projects ON DELETE CASCADE,
+     name TEXT NOT NULL,
```

Comment:
Would you make release name unique per project?

Reply:
I think yes; in case we decide to allow duplicate names, we can always drop the constraint, but in the opposite direction would be much more difficult.
Also, I'd probably name it `release_title`, if it better reflects the role of this field.